apps/src/templates/ProfessionalLearningApplyBanner.jsx

```diff
@@ -126,83 +25,136 @@ class ProfessionalLearningApplyBanner extends React.Component {
   }
 
   getText() {
-    if (this.props.useSignUpText) {
-      return 'Professional Learning is still on! Applications are open for Middle and High School teachers!';
+    if (this.props.customText) {
+      return this.props.customText;
     } else if (this.props.nominated) {
       return 'CONGRATULATIONS! You’ve been nominated for a scholarship!';
     } else if (this.props.teacherApplicationMode === 'closing-soon') {
-      return 'Applications closing soon! 2021 Professional Learning for Middle and High School teachers';
+      return 'Applications closing soon! 2022 Professional Learning for Middle and High School teachers';
     } else {
-      return 'SEATS OPEN - 2021 Professional Learning for Middle and High School teachers';
+      return 'SEATS OPEN - 2022 Professional Learning for Middle and High School teachers';
```

Comment:
I assume this is all currently English-only because our PD is English-only?

Reply:
Yes, it will only be shown in the US https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml#L13